tests/pos/i6375.scala

```diff
@@ -0,0 +1,38 @@
+/* In the following we should never have two nested closures after phase betaReduce
```

Comment:
We need a regression test for this in https://github.com/lampepfl/dotty/blob/master/compiler/test/dotty/tools/backend/jvm/InlineBytecodeTests.scala

Reply:
Yes, but I won't be the one adding it. Who can take care of this?

Reply:
Done.

@odersky could you double-check the bytecode to see it looks like what you expected.